src/modules/esl-utils/async/debounce.ts

```diff
@@ -23,7 +23,7 @@ export function debounce<F extends AnyToAnyFnSignature>(fn: F, wait = 10): Debou
   let timeout: number | null = null;
   let deferred: Deferred<ReturnType<F>> | null = null;
 
-  function debouncedSubject(...args: any[]) {
+  function debouncedSubject(...args: any[]){
```

Comment:
(Minor) ES Lint: missing formatting
merge actual main-beta state to have up to date es-lint rules